### ISSUE TITLE
[Attention] Information leak in visualization

### ISSUE THREAD

**Issue by tonysy** [pengzhiliang/MAE-pytorch#40]
God job.

Hi, I think this operation will leak the information of the original input image(mean and var of one patch). 
https://github.com/pengzhiliang/MAE-pytorch/blob/3546179ac5c4f0b80918b79f68678a42c8719808/run_mae_vis.py#L124

Anyone who uses the model trained with the normalized loss for visualization should pay attention to this operation.

I also suggest the author add the comment on this line. @pengzhiliang

**Comment by pengzhiliang** [comment]
Thank you! You are right.

Because the target in pre-training process is normalized, so the predict of model is unreal.
To visualize the reconstruction image, we add the predict and the original mean and var of each patch.

So, to avoid it, you need to use the real pixels as the target by setting `--normlize_target` to False. 

In fact, I am not sure that the reconstruction images shown in the paper from what kind of supervision.

And I will add the comment to avoid misunderstanding.

**Comment by MingfangDeng** [comment]
Dear author,
where the "--normlize_target"？I cannot find it. I hope you can say specificly,thank you.

**Comment by MingfangDeng** [comment]
OK,I find it，sorry to interrupt you.
